core/controllers/blog_admin.py

```diff
@@ -112,13 +112,13 @@ def post(self):
         result = {}
         if self.normalized_payload.get(
                 'action') == 'save_config_properties':
-            new_config_property_values = self.normalized_payload.get(
+            new_config_property_value_dicts = self.normalized_payload.get(
                 'new_config_property_values')
```

Comment:
The key name here should be new_config_property_value_dicts

Reply:
I have used ```new_config_property_values_dict``` since there is a single dict which contains several property values. 
Thank you